index.js

```diff
@@ -14,13 +14,14 @@ const convertWithOptions = (document, format, filter, options, callback) => {
     const installDir = tmp.dirSync({prefix: 'soffice', unsafeCleanup: true, ...tmpOptions});
     return async.auto({
         soffice: (callback) => {
-            let paths = [];
+            let paths = (options || {}).sofficeBinaryPaths ?? [];
```

Comment:
Not working fine.

Getting following error, while building the node app:
UnhandledPromiseRejectionWarning: /home/ubuntu/app-be/node_modules/libreoffice-convert/index.js:17
            let paths = (options || {}).sofficeBinaryPaths ?? [];

Reply:
Hmm, I tried all these values for options:

```
// const options = {sofficeBinaryPaths: []}
// const options = {}
const options = {sofficeBinaryPaths: ['test-path']}
const paths = (options || {}).sofficeBinaryPaths ?? []

console.log('paths', paths)
```

And each time it logs the expected value (either `[]` or `['test-path']`).  I'm not sure why this is happening for you.

What is the value of `options` you are passing in? Also, could you please provide the code snippet throwing this error?